package.json

```diff
@@ -65,9 +66,11 @@
     "- format": "# ------------------------",
     "format:js": "./scripts/format-project-js",
     "- e2e": "# ---------------------------",
-    "e2e:client": "npx cypress run",
-    "e2e:server": "npx cypress run --config baseUrl=http://localhost:1337",
-    "e2e:gui": "npx cypress open"
+    "e2e:client": "cypress run",
+    "e2e:server": "NODE_ENV=production cypress run --config baseUrl=http://localhost:1337",
```

Comment:
This ensures that if you run Cypress against the prod build it sets NODE_ENV internally so that the Vue Devtools test works properly.

Reply:
Would love to discuss this further. Some weird consequences of node env settings I want to touch base on. In particular, currently, App.vue() adds the root Vue instance to the window object during testing (when `if(winddow.Cypress) {}...`) without checking node env, but I'd been planning to only expose the app if not in prod, which would break many of the tests, as they rely heavily on access to the vuex store through the root application instance